generator/explores/glean_ping_explore.py

```diff
@@ -10,9 +11,43 @@ class GleanPingExplore(PingExplore):
 
     type: str = "glean_ping_explore"
 
+    def _to_lookml(self) -> dict:
+        """Generate LookML to represent this explore."""
+        views_lookml = self.get_view_lookml(self.views["base_view"])
+
+        # The first view, by convention, is always the base view with the
+        # majority of the dimensions from the top level.
+        base = views_lookml["views"][0]
+        base_name = base["name"]
+
+        joins = []
+        for view in views_lookml["views"][1:]:
+            view_name = view["name"]
+            metric = "__".join(view["name"].split("__")[1:])
+            join = {
+                "name": view["name"],
+                "relationship": "one_to_many",
+                "sql": f"CROSS JOIN UNNEST(${{{base_name}.{metric}}}) AS {view_name}",
+            }
+            joins.append(join)
+
+        return {
+            "name": self.name,
+            "view_name": self.views["base_view"],
+            "always_filter": {
+                "filters": self.get_required_filters("base_view"),
+            },
+            "joins": joins,
+        }
+
     @staticmethod
     def from_views(views: List[View]) -> Iterator[PingExplore]:
         """Generate all possible GleanPingExplores from the views."""
         for view in views:
             if view.view_type == GleanPingView.type:
                 yield GleanPingExplore(view.name, {"base_view": view.name})
+
+    @staticmethod
+    def from_dict(name: str, defn: dict, views_path: Path) -> PingExplore:
+        """Get an instance of this explore from a name and dictionary definition."""
+        return GleanPingExplore(name, defn["views"], views_path)
```

Comment:
This method wasn't implemented, which was causing it to return an instance of PingExplore instead. Took a while to debug.